src/middleware/issueDetails.middleware.js

```diff
@@ -1,128 +1,50 @@
-import performanceDbApi from '../services/performanceDbApi.js'
-import logger from '../utils/logger.js'
-import { types } from '../utils/logging.js'
-import { fetchDatasetInfo, fetchEntityCount, fetchLatestResource, fetchOrgInfo, isResourceIdInParams, logPageError, takeResourceIdFromParams, validateQueryParams } from './common.middleware.js'
+import {
+  addIssuesToEntities,
+  createPaginationTemplateParams,
+  extractJsonFieldFromEntities,
+  fetchActiveResourcesForOrganisationAndDataset,
+  fetchDatasetInfo,
+  fetchEntitiesFromIssuesWithReferences,
+  fetchEntityCount,
+  fetchIssueEntitiesCount,
+  fetchIssuesWithoutReferences,
+  fetchIssuesWithReferencesFromResourcesDatasetIssuetypefield,
+  fetchLatestResource,
+  fetchOrgInfo,
+  fetchSpecification,
+  formatErrorSummaryParams,
+  hasEntities,
+  isResourceIdNotInParams,
+  logPageError,
+  nestEntityFields,
+  pullOutDatasetSpecification,
+  replaceUnderscoreWithHyphenForEntities,
+  takeResourceIdFromParams,
+  validateQueryParams
+} from './common.middleware.js'
 import { fetchIf, renderTemplate } from './middleware.builders.js'
 import * as v from 'valibot'
-import { pagination } from '../utils/pagination.js'
 
-export const IssueDetailsQueryParams = v.object({
+export const IssueDetailsQueryParams = v.strictObject({
   lpa: v.string(),
   dataset: v.string(),
   issue_type: v.string(),
   issue_field: v.string(),
-  pageNumber: v.optional(v.string()),
+  pageNumber: v.string(),
```

Comment:
We can be a bit more strict here, e.g. `v.pipe(v.number(), v.integer(), v.toMinValue(0));`